[2014-05-16 08:22:36] log.ERROR: exception 'Illuminate\Database\Eloquent\ModelNotFoundException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php:121
Stack trace:
#0 /Users/programmer/sites/ace/app/routes.php(358): Illuminate\Database\Eloquent\Builder->firstOrFail()
#1 [internal function]: {closure}('solutions')
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array)
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable()
#4 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#5 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#6 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#7 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#8 {main} [] []
[2014-05-16 08:38:24] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 08:38:24] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 08:38:42] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 08:38:42] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 08:40:55] log.ERROR: exception 'Illuminate\Database\Eloquent\ModelNotFoundException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php:121
Stack trace:
#0 /Users/programmer/sites/ace/app/routes.php(358): Illuminate\Database\Eloquent\Builder->firstOrFail()
#1 [internal function]: {closure}('solutions')
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array)
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable()
#4 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#5 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#6 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#7 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#8 {main} [] []
[2014-05-16 08:41:04] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 08:41:12] log.ERROR: exception 'Illuminate\Database\Eloquent\ModelNotFoundException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php:121
Stack trace:
#0 /Users/programmer/sites/ace/app/routes.php(358): Illuminate\Database\Eloquent\Builder->firstOrFail()
#1 [internal function]: {closure}('solutions')
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array)
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable()
#4 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#5 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#6 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#7 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#8 {main} [] []
[2014-05-16 08:41:45] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 08:41:45] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 08:41:54] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 09:03:15] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 09:22:49] log.ERROR: exception 'ErrorException' with message 'Undefined offset: 1' in /Users/programmer/sites/ace/app/libraries/helper.php:2004
Stack trace:
#0 /Users/programmer/sites/ace/app/libraries/helper.php(2004): Illuminate\Exception\Handler->handleError(8, 'Undefined offse...', '/Users/programm...', 2004, Array)
#1 /Users/programmer/sites/ace/app/storage/views/c1b86bf070fcea1991b4ac7f09c8a79d(9): the_content('<p>Lorem ipsum ...')
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/Users/programm...')
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/Users/programm...', Array)
#4 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/Users/programm...', Array)
#5 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents()
#6 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render()
#7 /Users/programmer/sites/ace/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#8 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#9 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#10 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#11 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#12 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#13 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#14 {main} [] []
[2014-05-16 09:23:39] log.ERROR: exception 'ErrorException' with message 'Undefined variable: short_code' in /Users/programmer/sites/ace/app/libraries/helper.php:2009
Stack trace:
#0 /Users/programmer/sites/ace/app/libraries/helper.php(2009): Illuminate\Exception\Handler->handleError(8, 'Undefined varia...', '/Users/programm...', 2009, Array)
#1 /Users/programmer/sites/ace/app/storage/views/c1b86bf070fcea1991b4ac7f09c8a79d(9): the_content('<p>Lorem ipsum ...')
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/Users/programm...')
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/Users/programm...', Array)
#4 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/Users/programm...', Array)
#5 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents()
#6 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render()
#7 /Users/programmer/sites/ace/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#8 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#9 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#10 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#11 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#12 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#13 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#14 {main} [] []
[2014-05-16 09:23:53] log.ERROR: exception 'ErrorException' with message 'Uninitialized string offset: 0' in /Users/programmer/sites/ace/app/libraries/helper.php:2019
Stack trace:
#0 /Users/programmer/sites/ace/app/libraries/helper.php(2019): Illuminate\Exception\Handler->handleError(8, 'Uninitialized s...', '/Users/programm...', 2019, Array)
#1 /Users/programmer/sites/ace/app/storage/views/c1b86bf070fcea1991b4ac7f09c8a79d(9): the_content('<p>Lorem ipsum ...')
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/View/Engines/PhpEngine.php(38): include('/Users/programm...')
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/View/Engines/CompilerEngine.php(45): Illuminate\View\Engines\PhpEngine->evaluatePath('/Users/programm...', Array)
#4 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/View/View.php(100): Illuminate\View\Engines\CompilerEngine->get('/Users/programm...', Array)
#5 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/View/View.php(81): Illuminate\View\View->getContents()
#6 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Http/Response.php(70): Illuminate\View\View->render()
#7 /Users/programmer/sites/ace/vendor/symfony/http-foundation/Symfony/Component/HttpFoundation/Response.php(141): Illuminate\Http\Response->setContent(Object(Illuminate\View\View))
#8 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1414): Symfony\Component\HttpFoundation\Response->__construct(Object(Illuminate\View\View))
#9 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Route.php(58): Illuminate\Routing\Router->prepare(Object(Illuminate\View\View), Object(Illuminate\Http\Request))
#10 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#11 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#12 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#13 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#14 {main} [] []
[2014-05-16 09:24:47] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 09:24:58] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 09:35:58] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 09:45:13] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 09:45:20] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 09:45:20] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 09:45:43] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 09:45:44] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 09:46:22] log.ERROR: exception 'Illuminate\Database\Eloquent\ModelNotFoundException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php:121
Stack trace:
#0 /Users/programmer/sites/ace/app/routes.php(358): Illuminate\Database\Eloquent\Builder->firstOrFail()
#1 [internal function]: {closure}('contact')
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array)
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable()
#4 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#5 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#6 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#7 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#8 {main} [] []
[2014-05-16 09:47:41] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 09:48:17] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 09:48:30] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:09:20] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:09:25] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:09:25] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:10:57] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:11:07] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:11:24] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:11:29] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:11:29] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:11:41] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:12:09] log.ERROR: exception 'Illuminate\Database\Eloquent\ModelNotFoundException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php:121
Stack trace:
#0 /Users/programmer/sites/ace/app/routes.php(358): Illuminate\Database\Eloquent\Builder->firstOrFail()
#1 [internal function]: {closure}('projects')
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array)
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable()
#4 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#5 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#6 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#7 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#8 {main} [] []
[2014-05-16 10:12:11] log.ERROR: exception 'Illuminate\Database\Eloquent\ModelNotFoundException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php:121
Stack trace:
#0 /Users/programmer/sites/ace/app/routes.php(358): Illuminate\Database\Eloquent\Builder->firstOrFail()
#1 [internal function]: {closure}('projects')
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array)
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable()
#4 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#5 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#6 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#7 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#8 {main} [] []
[2014-05-16 10:12:23] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:12:26] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:12:27] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:13:19] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:13:36] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:13:36] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:13:45] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:14:03] log.ERROR: exception 'Illuminate\Database\Eloquent\ModelNotFoundException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php:121
Stack trace:
#0 /Users/programmer/sites/ace/app/routes.php(358): Illuminate\Database\Eloquent\Builder->firstOrFail()
#1 [internal function]: {closure}('company')
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array)
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable()
#4 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#5 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#6 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#7 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#8 {main} [] []
[2014-05-16 10:14:09] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:14:23] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:15:33] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:15:39] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:15:39] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:15:51] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:26:21] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:26:47] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:27:56] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:28:07] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:28:27] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:28:32] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:28:45] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:29:45] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:29:50] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:30:10] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:30:11] log.ERROR: exception 'Illuminate\Database\Eloquent\ModelNotFoundException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php:121
Stack trace:
#0 /Users/programmer/sites/ace/app/routes.php(358): Illuminate\Database\Eloquent\Builder->firstOrFail()
#1 [internal function]: {closure}('draft')
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Route.php(80): call_user_func_array(Object(Closure), Array)
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Route.php(47): Illuminate\Routing\Route->callCallable()
#4 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1016): Illuminate\Routing\Route->run(Object(Illuminate\Http\Request))
#5 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#6 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#7 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#8 {main} [] []
[2014-05-16 10:31:17] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
[2014-05-16 10:31:37] log.ERROR: exception 'Symfony\Component\HttpKernel\Exception\NotFoundHttpException' in /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php:1429
Stack trace:
#0 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1050): Illuminate\Routing\Router->handleRoutingException(Object(Symfony\Component\Routing\Exception\ResourceNotFoundException))
#1 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Routing/Router.php(1014): Illuminate\Routing\Router->findRoute(Object(Illuminate\Http\Request))
#2 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(530): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#3 /Users/programmer/sites/ace/vendor/laravel/framework/src/Illuminate/Foundation/Application.php(506): Illuminate\Foundation\Application->dispatch(Object(Illuminate\Http\Request))
#4 /Users/programmer/sites/ace/index.php(50): Illuminate\Foundation\Application->run()
#5 {main} [] []
